.github/workflows/release_win.yml

```diff
@@ -100,7 +100,7 @@ jobs:
       if: ${{ always() }}
       run: |
         cd onnx
-        python -m pip uninstall -y numpy onnx && python -m pip install numpy
+        python -m pip uninstall -y numpy onnx && python -m pip install "numpy<2"
```

Comment:
name should be updated to add "(below version2)" like the others.

Reply:
thanks